pkg/net/libp2p/unicast_channel_test.go

```diff
@@ -63,6 +63,9 @@ func TestSendReceiveUnicastChannel(t *testing.T) {
 		}
 
 		// Peer 2 get the channel from cache.
+		// Wait for a moment until the messages sent by peer 1 are received by
+		// peer 2, so that the channel is opened and unmarshaller set.
+		time.Sleep(2 * time.Second)
```

Comment:
Why this is exactly 2 seconds?

Reply:
I think another value like 1 s 0r 0,5 s would work as well.
I chose 2 seconds because we already had such a sleep time in `unicast_channel_test.go` and it worked well.